### PR TITLE
Improve user management documentation

### PR DESCRIPTION
**Pull request by cccs-eric**
This PR is to address a [Copilot issue](https://cccs.atlassian.net/browse/DOBBY-151).  The JIRA ticket contains the details and the play-by-play.

### CODE REVIEW

File: docs/administration/user_management.md [hunk 1]
```diff
@@ -1,50 +1,87 @@
...
-6. Change the users' roles, quotas, and passwords
+## Accessing the User Management Interface
 
-You can find the user management interface by clicking the *Administration* topic then choose the *System Safelist* subtopic.
+To begin managing user accounts in Assemblyline, follow these steps to access the user management interface:
 
-![User management](./images/users_bar.png){: .center }
+1. Click on the **Administration** topic in your Assemblyline interface.
+2. Under the Administration section, select the **System Safelist** subtopic.
+3. Once in the System Safelist, you will be presented with the user management interface.
```
Comment (cccs-rs) [inline docs/administration/user_management.md:11]:
```suggestion
2. Under the Administration section, select the **Users** subtopic.
3. Once in the Users, you will be presented with the user management interface.
```

File: docs/administration/user_management.md [hunk 2]
```diff
@@ -1,50 +1,87 @@
...
-1. Remove it
-2. Enable/disable access its access to the system
-3. Set its roles
-4. Set its quotas
-5. Change its password
+On the user detail page, you can take various administrative actions:
+
+- Remove the user account entirely.
+- Enable or disable the user's access to Assemblyline.
+- Adjust the user's role settings as appropriate.
```
Comment (cccs-rs) [inline docs/administration/user_management.md:45]:
```suggestion
- Adjust the user's role settings or classification as appropriate.
```

File: docs/administration/user_management.md [hunk 3]
```diff
@@ -1,50 +1,87 @@
...
+To view further details about a particular user, simply click on their username in the user list. This will open the user detail page, which provides comprehensive information about the user, including:
 
-1. Identity of the user (username, avatar, name, groups, email address)
-2. Roles of the user
-3. Allowed quotas for the user
-4. Status of the user
+![User Detail](./images/user_detail.png)
 
-From this page, you will be able to take the following actions on the current user:
+- Profile: Including username, avatar, full name, groups, and email address.
```
[the remark this answers is not in the document]
Comment (cccs-eric) [reply docs/administration/user_management.md:36]: Is this the "classification of the user", or the "classification of that UI page"?  If it is about the page, I don't think it should be mentioned in the `Profile`.

File: docs/administration/user_management.md [hunk 4]
```diff
@@ -1,50 +1,87 @@
...
+- Change the user's password securely.
+
+### Step-by-Step Actions on User Detail Page
+
+#### Enabling or Disabling a User
+
+To change a user's access to the system:
+
+1. Navigate to the *Status* section of the user's detail page.
+2. Click the **Enable** or **Disable** button, depending on the desired action.
```
Comment (cccs-rs) [inline docs/administration/user_management.md:56]:
```suggestion
1. Click the **Account Enabled** or **Account Disabled** button, depending on the desired action, located below the avatar in the user's detail page.
2. Once you're satisfied with the change, you can click on the "Save Changes" button located at the bottom of the window
```

File: docs/administration/user_management.md [hunk 5]
```diff
@@ -1,50 +1,87 @@
...
+To update a user's roles:
+
+1. Locate the *Roles* section within the detail page.
+2. Select or unselect roles as needed.
+
+#### Adjusting User Quotas
+
+To set or change usage quotas:
+
+1. Go to the *Quotas* section.
```
Comment (cccs-rs) [inline docs/administration/user_management.md:69]:
```suggestion
1. Go to the relevant quota setting in the *Options* section.
```

File: docs/administration/user_management.md [hunk 6]
```diff
@@ -1,50 +1,87 @@
...
+2. Input the new quota values as per your requirements.
+
+#### Updating User Password
+
+To change a user's password:
+
+1. In the *Password* section, click on the **Change Password** button.
```
Comment (cccs-rs) [inline docs/administration/user_management.md:76]:
```suggestion
1. In the *Security* section, click on the **Change Password** button.
```

File: docs/administration/user_management.md [hunk 7]
```diff
@@ -1,50 +1,87 @@
...
+2. Follow the prompts to enter and confirm the new password.
+
+### Removing a User
+
+To permanently remove a user account from the system:
+
+1. Navigate to the in-detail user page.
+2. From the options, select the **Remove** action.
+3. Confirm the removal when prompted.
```
Comment (cccs-rs) [inline docs/administration/user_management.md:85]:
```suggestion
1. Select the *Remove* action located above the avatar in the user's detail page
2. Confirm the removal when prompted.
```

File: docs/administration/user_management.md [hunk 8]
```diff
@@ -1,50 +1,87 @@
...
+
+By following the detailed steps outlined in this guide, you can effectively manage user accounts within Assemblyline to ensure a secure and streamlined operation.
```
Comment (cccs-rs) [inline docs/administration/user_management.md:87]:
```suggestion

To apply any changes to the user account, you'll need to select the **Save changes** button located at the bottom of the window.

By following the detailed steps outlined in this guide, you can effectively manage user accounts within Assemblyline to ensure a secure and streamlined operation.
```

File: docs/administration/user_management.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (cccs-rs) [reply docs/administration/user_management.md:36]: Classification of the user, in terms of what they have access to.

[the remark this answers is not in the document]
Comment (cccs-eric) [reply docs/administration/user_management.md:36]: What about the following?

> Profile: Including username, avatar, full name, groups, email address, and, if enabled by the administrator, security classification. The user's security classification dictates the level of information the user is permitted to access within the system.

**Pull request merged**